src/App.js

```diff
@@ -1,16 +1,26 @@
 import React from 'react';
+import { useNavigate } from 'react-router-dom';
 import './App.css';
 import CreditCard from './CreditCard';
 import ErrorName from './MessagesError/ErrorName';
 import ErrorNumber from './MessagesError/ErrorNumber';
 import ErrorCode from './MessagesError/ErrorCode';
 import ErrorDate from './MessagesError/ErrorDate';
+import Completed from './MessagesCard/Completed';
+import Incompleted from './MessagesCard/Incompleted';
 
 function App() {
+
+  const Back = useNavigate()
+
+  function BackShoppingCar(){
+    Back('/')
+  }
+
   return (
     <div className="App">
       <header>
-        <span>&#8249;</span>
+        <span onClick={BackShoppingCar}>&#8249;</span>
```

Comment:
I think a more "natural" way to do this is using the Link component

https://reactrouter.com/en/main/components/link

```js
<Link to="/">&#8249;</Link>
```